### PR TITLE
Adding mailgun sample

### PR DESCRIPTION
**Pull request by jonparrott**

### CODE REVIEW

File: appengine/mailgun/appengine_config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from google.appengine.ext import vendor
```
Comment (waprin) [inline appengine/mailgun/appengine_config.py]: license? technically code so I think it needs it.

File: appengine/mailgun/main_test.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        with http:
+            response = self.app.post('/', {
+                'recipient': 'jonwayne@google.com',
+                'submit': 'Send simple email'})
+
+            self.assertEqual(response.status_int, 200)
+
+        http = Http2Mock(responses=[{}])
+
+        with http:
```
Comment (waprin) [inline appengine/mailgun/main_test.py]: could assert http was called, check the data it was called with matches what's expected. That's probably more than necessary, just food for thought.

File: tests/utils.py [hunk 3]
```diff
@@ -21,6 +21,7 @@
 import tempfile
 import unittest
```
Comment (waprin) [inline tests/utils.py]: license?

**Comment by waprin** [comment]
LGTM, think you're missing one license file.

File: appengine/mailgun/main_test.py (hunk 2 above)
Comment (jonparrott) [inline appengine/mailgun/main_test.py]: Coverage handles that.

Comment (waprin) [inline appengine/mailgun/main_test.py]: Gotcha

**Comment by jonparrott** [comment]
Fixed missing license. Merging.

**Pull request merged**